index.js

```diff
@@ -84,19 +86,85 @@ module.exports = {
     }
   },
 
+  _shouldIncludeHelpers() {
+    let customAddonOptions = this.parent && this.parent.options && this.parent.options['ember-cli-babel'];
+
+    if (customAddonOptions && 'includeExternalHelpers' in customAddonOptions) {
+      throw new Error('includeExternalHelpers is not supported in addon configurations, it is an app-wide configuration option');
+    }
+
+    let hostOptions = this._getHostOptions();
+    let customOptions = hostOptions['ember-cli-babel'];
+
+    if (customOptions && 'includeExternalHelpers' in customOptions) {
+      return customOptions.includeExternalHelpers === true;
+    } else {
+      // Check the project to see if we should include helpers based on heuristics.
+      return defaultShouldIncludeHelpers(this.project);
+    }
+  },
+
+  _getHelperVersion() {
+    if (!this._helperVersion) {
+      let checker = new VersionChecker(this.project);
+      this._helperVersion = checker.for('@babel/core', 'npm').version;
+    }
+
+    return this._helperVersion;
+  },
+
+  _getHelpersPlugin() {
+    return [
+      [
+        require.resolve('@babel/plugin-transform-runtime'),
+        {
+          version: this._getHelperVersion(),
```

Comment:
I see how they're using `version` in the implementation, but I can't find any documentation for it on [the plugin page](https://babeljs.io/docs/en/babel-plugin-transform-runtime). I don't imagine that's intended to be a "secret" option, but have you seen anything indicating it's public?

Reply:
I have not, but it's definitely needed to actually include decorator helpers or anything more recent than 7.0.0-0, so I can't imagine it's not public? Does seem very strange to me though